draft-ietf-webpush-protocol.xml

```diff
@@ -1270,16 +1261,23 @@ HEADERS      [stream 82] +END_STREAM
 
      <section title="Denial of Service Considerations">
        <t>
-         Discarding unwanted messages at the user agent based on message
-         authentication doesn't protect against a denial of service attack on
-         the user agent.  Even a relatively small volume of push messages can
-         cause battery-powered devices to exhaust power reserves.
+         End-to-end confidentiality mechanisms, such as those in <xref
+         target="API"/>, prevent an entity with a valid push message
+         subscription URI from learning the contents of push messages.  Push

```

Comment:
The problem here isn't that a leak of the URI allows an attacker to read the messages, for DoS, the main consideration is whether they can *send* a message.